### PR TITLE
[Doc] Add Studio doc

### PR DESCRIPTION
**Pull request by gaoyan1998**
## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

File: docs/docs/user_guide/studio/flink_sql_task_devlop.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+![image-20231220114448998](http://pic.dinky.org.cn/dinky/docs/test/202312201144048.png)
+
+### 版本历史
+
+在创建作业后，点击`发布`会自动创建一个历史版本，用于记录历史并回退
+
+![image-20231220134050332](http://pic.dinky.org.cn/dinky/docs/test/202312201340401.png)
+
+单机历史版本即可查看当前版本与所选版本的代码差异
```
Comment (Zzm0809) [inline docs/docs/user_guide/studio/flink_sql_task_devlop.md]: 单击

File: docs/docs/user_guide/studio/flink_sql_task_devlop.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+       name
+from datagen_source;
+```
+
+点击提交按钮，即可提交任务到集群，任务提交完成，我们可以进入运维中心页面，状态为Running表示运行成功。
+![](http://pic.dinky.org.cn/dinky/docs/zh-CN//fast-guide-devops.png)
+
+## Flink JAR作业
+
+编写FlinkSql语句
```
Comment (Zzm0809) [inline docs/docs/user_guide/studio/flink_sql_task_devlop.md:150]: 加一下注意事项 此为dinky扩展语法

File: docs/docs/user_guide/studio/flink_sql_task_devlop.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+'parallelism'='',
+);
+```
+
+:::warning 注意
+
+1. 以上示例中, uri 参数使用了 rs 协议, 请参考 [资源管理](../../user_guide/register_center/resource) 中 rs 协议使用方式
+2. 以上示例中, uri 的值为 rs:/jar/flink/demo/SocketWindowWordCount.jar, 该值为资源中心中的资源路径,
+   请确保资源中心中存在该资源,请忽略资源中心 Root 节点(该节点为虚拟节点)
+3. 目前仅支持 rs 与 file 协议头，后续会支持更多协议，请关注版本更新
```
Comment (Zzm0809) [inline docs/docs/user_guide/studio/flink_sql_task_devlop.md]: rs可以直接桥接所有资源中心配置的协议

File: docs/docs/user_guide/studio/studio_overview.md [hunk 4]
```diff
@@ -20,4 +20,40 @@ Dinky 数据开发功能主要包括：
...
+
+![image-20231220140415854](http://pic.dinky.org.cn/dinky/docs/test/202312201404899.png)
+
+#### FlinkSql作业
+
+此作业类型用于开发**FlinkSQL**，与 **FlinkCDC整库同步**作业。
+
+#### FlinkJar作业
+
+用于运行**自定义jar包**，对于非Flink sql作业，使用原生flink代码开发的jar包，可以通过dinky的`exec jar`语法进行提交与管理
```
Comment (Zzm0809) [inline docs/docs/user_guide/studio/studio_overview.md]: execute jar

**Review by aiwenmo — APPROVED** [review]
LGTM

**Review by leechor — COMMENTED** [review]
haha

File: docs/docs/get_started/overview.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+id: overview
+title: Dinky介绍
+---------
+
+## 概述
+
+实时即未来，Dinky 为 Apache Flink 而生，让 Flink SQL 纵享丝滑。
+
+Dinky 是一个开箱即用、易扩展，以 Apache Flink 为基础，连接 OLAP 和数据湖等众多框架的一站式实时计算平台，致力于流批一体和湖仓一体的探索与实践。
+Dinky 是一个开箱即用、易扩展，以 Apache Flink 为基础，连接 OLAP 和数据湖等众多框架的一站式实时计算平台，致力于流批一体和湖仓一体的探索与实践，
```
Comment (Zzm0809) [inline docs/docs/get_started/overview.md]: 重复了

**Review by Zzm0809 — APPROVED** [review]
LGTM

**Pull request merged**